langchain/src/tools/gmail/create_draft.ts

```diff
@@ -0,0 +1,71 @@
+import { google } from "googleapis";
+import { encodeToBase64 } from "@gomomento/sdk-core/dist/src/internal/utils/string.js";
+import { GmailBaseTool, GmailBaseToolParams } from "./base.js";
+
+export interface CreateDraftSchema {
+  message: string;
+  to: string[];
+  subject: string;
+  cc?: string[];
+  bcc?: string[];
+}
+
+export class GmailCreateDraft
+  extends GmailBaseTool
```

Comment:
whys this on a new line? did the linter do that? the line doesnt look that long to me